File: app/src/main/java/com/mmjang/ankihelper/data/dict/MaldpeDatabaseHelper.java
package com.mmjang.ankihelper.data.dict;

import android.content.Context;

public class MaldpeDatabaseHelper extends BaseDatabaseHelper {

    private static final String DATABASE_NAME = "maldpe.db";

    public MaldpeDatabaseHelper(Context context) {
        super(context, DATABASE_NAME, 1);
    }
}

File: app/src/main/java/com/lmyby/ankiquicker/data/dict/MaldpeDatabaseHelper.kt
package com.lmyby.ankiquicker.data.dict

import android.content.Context

/**
 * Converted to Kotlin as part of Phase 2 data model migration
 */
class MaldpeDatabaseHelper(context: Context) : BaseDatabaseHelper(context, DATABASE_NAME, 1) {
    companion object {
        private const val DATABASE_NAME = "maldpe.db"
    }
}
